src/_data/sidenav.yml

```diff
@@ -38,10 +38,12 @@
       permalink: https://github.com/flutter/flutter/tree/master/examples/flutter_gallery#flutter-gallery
 
 - title: Development
+  permalink: /development
   children:
     - title: User interface
+      permalink: /development/ui
       children:
-        - title: Tour the framework
+        - title: Introduction to Widgets
```

Comment:
@Sfshaza - are you ok with this change in title. That page wasn't really a tour, it was (as the file name implied), an introduction. cc @filiph

Reply:
Yes.